src/assets/translations/en/main.json

```diff
@@ -368,7 +368,7 @@
     "tooltip": {
       "rate": "This is the expected rate for this trade pair.",
       "minerFee": "This is the fee to process the transaction on the blockchain, this is not a fee charged by ShapeShift.",
-      "shapeshiftFee": "ShapeShift adds no fees on trades 🎉"
+      "shapeshiftFee": "ShapeShift DOES NOT add fees on trades 🎉"
```

Comment:
Is there a product spec for this new copy? I wasn't able to find it on the linked issue

Reply:
@gomesalexandre No, there is no specific copy from product that we have. Just to be explicit that ShapeShift does not take any fees on trading.

Reply:
@shapeshift/product thoughts?

Reply:
new diff is good per @DiggyDiggy2

Reply:
@DaoDev44 plz don't capitalize